src/main/java/seedu/address/model/petpatient/PetPatient.java

```diff
@@ -8,7 +8,8 @@
 import java.util.Optional;
 import java.util.Set;
 
-import seedu.address.model.person.Person;
+//import seedu.address.commons.events.ui.NewResultAvailableEvent;
+import seedu.address.model.person.Nric;
```

Comment:
Checkstyle error: 'seedu.address.model.person.Nric' should be separated from previous import group by one line. [CustomImportOrder]